testresources/__init__.py

```diff
@@ -410,6 +418,13 @@ def _makeOrder(self, partition):
         return order[1:]
 
 
+OptimisingTestSuite.known_suite_classes = (
+    unittest.TestSuite, OptimisingTestSuite)
+if unittest2 is not None:
+    OptimisingTestSuite.known_suite_classes = \
+        OptimisingTestSuite.known_suite_classes + (unittest2.TestSuite,)
```

Comment:
Is there a reason not to do:

OptimisingTestSuite.known_suite_classes += (unittest2.TestSuite,)

?

What you have is fine, but it's more typing, and forces a line-break (maybe my suggestion would wrap lines as well, I haven't counted the characters).